Sources/FluentUI_iOS/Components/List/FluentListSectionHeader.swift

```diff
@@ -44,16 +57,37 @@ public struct FluentListSectionHeader<Title: StringProtocol, TrailingContent: Vi
         return contentView
     }
 
+    // MARK: private methods
+
+    static func tokenSetStyle(_ style: ListSectionHeaderStyle) -> TableViewHeaderFooterView.Style {
+        switch style {
+        case .regular:
+            return .header
+        case .primary:
+            return .headerPrimary
+        }
+    }
+
     // MARK: Private variables
 
     private var trailingContent: (() -> TrailingContent)?
     private let title: Title
+    private let tokenSet: TableViewHeaderFooterViewTokenSet
```

Comment:
similar to `ListItem`, should we have a `FluentListSectionHeaderTokenSet` alias

Reply:
Yeah makes sense, I can add!